packages/client/src/features/accessibility/toast/toast-tool.ts

```diff
@@ -0,0 +1,155 @@
+/********************************************************************************
+ * Copyright (c) 2023 Business Informatics Group (TU Wien) and others.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * This Source Code may also be made available under the following Secondary
+ * Licenses when the conditions for such availability set forth in the Eclipse
+ * Public License v. 2.0 are satisfied: GNU General Public License, version 2
+ * with the GNU Classpath Exception which is available at
+ * https://www.gnu.org/software/classpath/license.html.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+ ********************************************************************************/
+
+import { injectable, inject } from 'inversify';
+import { Action, AbstractUIExtension, IActionHandler, ICommand, TYPES } from '~glsp-sprotty';
+import { IDiagramStartup } from '../../../base/model/diagram-loader';
+import { GLSPActionDispatcher } from '../../../base/action-dispatcher';
+import { EditorContextService } from '../../../base/editor-context-service';
+
+export interface HideToastAction extends Action {
```

Comment:
Perhaps it would make sense to extract the action declarations into a separate file in this `toast` directory, wdyt?